Wordle.Api/Wordle.Api/Identity/Claims.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Wordle.Api.Identity;
+public class Claims
+{
+    public const string UserId = "userId";
+    public const string Random = "random";
+    public const string UserName = "userName";
```

Comment:
```suggestion
    public const string UserName = "userName";
    public const string Age = "age";
    public const string Birthdate = "birthdate";
    public const string MotU = "masterOfTheUniverse";
```

You can probably get away with only doing either age or birthday.